### ISSUE TITLE
bug: 🐝 Getting a file not found error

### ISSUE THREAD

**Issue by DevGranil** [JamesIves/github-pages-deploy-action#1698]
### Describe the bug

Hi James

Awesome tool! used on a few of my projects.

I'm getting the following issue when deploying 

File not found: '/home/runner/work/_actions/JamesIves/github-pages-deploy-action/v4/lib/main.js'

This used to work before - I've made no changes to my repo

Many thanks

### Reproduction Steps

make changes and push to repo 

### Workflow

```yml
name: Angular Build and Deploy

on:
  push:
    branches:
      - main

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - name: Checkout code
      uses: actions/checkout@v4

    - name: Setup Node.js
      uses: actions/setup-node@v4
      with: 
        node-version: 20

    - name: Build Angular app to prod
      working-directory: ./tic-tac-toe-app
      run: npm install && npm run build:prod

    - name: Run Tests
      working-directory: ./tic-tac-toe-app
      run: npm run test:headless

    - name: Deploy to Github Pages
      uses: JamesIves/github-pages-deploy-action@v4
      with:
        token: ${{ secrets.GH_TOKEN }}
        BRANCH: prod
        FOLDER: tic-tac-toe-app/dist/browser
```

**Comment by JulianYG** [comment]
+1 here

**Comment by chunlaw** [comment]
Same 🥹

`Run JamesIves/github-pages-deploy-action@v4
Error: File not found: '/home/runner/work/_actions/JamesIves/github-pages-deploy-action/v4/lib/main.js'`

**Comment by gismountains** [comment]
Workaround for now: https://github.com/JamesIves/github-pages-deploy-action/issues/1697#issuecomment-2379933098

**Comment by JamesIves** [comment]
This issue has now been resolved as of [v4.6.7](https://github.com/JamesIves/github-pages-deploy-action/releases/tag/v4.6.7), apologies about that. I'm going to make some adjustments to the integration tests so this doesn't go missed during the release cycle. The `v4` major tag has also been updated so any existing workflows should now work as expected.
